Dockerfile

```diff
@@ -15,9 +15,13 @@
 FROM golang:1.10.1-alpine3.7 as builder
 WORKDIR /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 ADD . .
-RUN CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o bin/gce-pd-csi-driver ./cmd/
+RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o bin/gce-pd-csi-driver ./cmd/
 
-FROM alpine:3.7
+# Start from Google Debian base
+FROM gcr.io/google-containers/debian-base-amd64:0.3
 COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/bin/gce-pd-csi-driver /gce-pd-csi-driver
 
-ENTRYPOINT ["/gce-pd-csi-driver"]
\ No newline at end of file
+# Install necessary dependencies
+RUN clean-install -y --no-install-recommends util-linux e2fsprogs mount ca-certificates
```

Comment:
you don't need the options, clean-install already runs with them
https://github.com/kubernetes/kubernetes/blob/master/build/debian-base/clean-install